### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (a[href="#"])

### ISSUE THREAD

**Issue by markreay** [ada-cat/accessibility#2]
**Issue**: `Elements must have sufficient color contrast` ([`color-contrast`](https://dequeuniversity.com/rules/axe/3.1/color-contrast?application=axeAPI))

**Target application**: [Accessible University Demo Site - Inaccessible Version](https://markreay.github.io/AU/before.html)

**Element path**: li:nth-child(4) > a[href="#"]

**Snippet**:

    <a href="#">Visitors</a>

**How to fix**:

    Fix any of the following:
      Element has insufficient color contrast of 2.52 (foreground color: #8a94a8, background color: #e7ecd8, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1

**Environment**:
Chrome version 72.0.3626.109

====

This accessibility issue was found using Accessibility Insights for Web 2.0.1 (axe-core 3.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
